frontend/src/components/custom-tools/prompt-card/PromptOutput.jsx

```diff
@@ -205,13 +204,18 @@ function PromptOutput({
               transition={{ duration: 0.5, ease: "linear" }}
               className={`prompt-card-llm ${
                 details?.enable_highlight &&
+                enforceType !== "json" &&
                 selectedHighlight?.highlightedPrompt === promptId &&
                 selectedHighlight?.highlightedProfile === profileId &&
                 "highlighted-prompt-cell"
               }`}
             >
               <Tooltip
-                title={details?.enable_highlight && "Click to highlight"}
+                title={
+                  details?.enable_highlight && enforceType !== "json"
+                    ? "Click to highlight"
+                    : "Highlight not supported for enforce type JSON"
```

Comment:
```suggestion
                    : "Highlight not supported for enforce type JSON"
```"Highlighting is not supported when enforce type is JSON"